scalafix-core/src/main/scala/scalafix/internal/config/ScalafixConfig.scala

```diff
@@ -34,33 +37,6 @@ object ScalafixConfig {
   implicit lazy val ScalafixConfigDecoder: ConfDecoder[ScalafixConfig] =
     decoder(default)
 
-  val DefaultDialect: Dialect = Scala212.copy(
-    // Are `&` intersection types supported by this dialect?
-    allowAndTypes = true,
-    // Are extractor varargs specified using ats, i.e. is `case Extractor(xs @ _*)` legal or not?
-    allowAtForExtractorVarargs = true,
-    // Are extractor varargs specified using colons, i.e. is `case Extractor(xs: _*)` legal or not?
-    allowColonForExtractorVarargs = true,
-    // Are inline vals and defs supported by this dialect?
-    allowInlineMods = false,
-    // Are literal types allowed, i.e. is `val a : 42 = 42` legal or not?
-    allowLiteralTypes = true,
-    // Are `|` (union types) supported by this dialect?
-    allowOrTypes = true,
-    // Are trailing commas allowed? SIP-27.
-    allowTrailingCommas = true,
-    // Are trait allowed to have parameters?
-    // They are in Dotty, but not in Scala 2.12 or older.
-    allowTraitParameters = true,
-    // Are view bounds supported by this dialect?
-    // Removed in Dotty.
-    allowViewBounds = true,
-    // Are `with` intersection types supported by this dialect?
-    allowWithTypes = true,
-    // Are XML literals supported by this dialect?
-    // We plan to deprecate XML literal syntax, and some dialects
-    // might go ahead and drop support completely.
-    allowXmlLiterals = true
-  )
-
+  val DefaultDialect = scala.meta.dialects.Scala213
```

Comment:
Following up on the previous discussion [above](https://github.com/scalacenter/scalafix/pull/1373/files#r614766745): this is a breaking change since currently some scala3 syntax can be parsed without tweaking the default. That's what we want but we should document that.

Reply:
I can keep the same if needed. I just need to check which one was actually used.

Reply:
so this default is based on 212 and it was only used in old rules (none of the ones we have in scalafix).
the real dialect that was used is this one, where all the additional configuration is already part of scala213.
```
  private val dialect = scala.meta.dialects.Scala213.copy(
    allowLiteralTypes = true,
    allowTrailingCommas = true,
    allowInlineMods = flase, 
    allowInlineIdents = true 
  )
  ```
  I don't know if it's useful to allow scala 3 syntax in scala213, but we could allow some of it.